packages/ui/src/lib/screen/EmptyScreen.svelte

```diff
@@ -21,10 +24,11 @@ onMount(() => {
   processed = true;
 });
 
-function copyRunInstructionToClipboard() {
+function copyRunInstructionToClipboard(): void {
   const text = copyTextDivElement?.textContent;
   if (text) {
-    window.clipboardWriteText(text);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (window as any).clipboardWriteText(text);
```

Comment:
I think this part is problematic as it's a custom API of Podman Desktop exposed to window object, but it won't work if you're like in a webview

Reply:
Should be solved now